README.md

```diff
@@ -37,13 +39,11 @@ Note that beanstalk apps may need to be "Restart"ed to pick up env config change
 After 1) PR signoff and 2) confirming that your changes don't create trouble for [nypl-core-objects](https://github.com/NYPL/nypl-core-objects) or other immediately impacted apps, proceed to publish.
 
 1. Merge your PR and delete feature branch.
-2. Update [README.md](README.md#current-version) with the new version number (e.g. "v1.33")
-3. Add an entry to [CHANGELOG.md](CHANGELOG.md) summarizing the changes
-4. Commit your changes
-5. Add a release tag (e.g. `v1.33`. See [Working with Git tags](#working-with-git-tags))
-6. Push to master
-7. Add release notes at https://github.com/NYPL/nypl-core/tags .
-8. If you made changes to `./vocabularies`, [publish nypl-core-objects lookups to "production" S3](https://github.com/NYPL/nypl-core-objects#pushing-to-s3)
+2. Commit your changes
+3. Add a release tag (e.g. `v1.33`. See [Working with Git tags](#working-with-git-tags))
+4. Push to master
+5. Add release notes at https://github.com/NYPL/nypl-core/tags .
```

Comment:
Is this process documented anywhere?

Reply:
If this is about "Add release notes at ...", I propose we retire that. It's not been kept up to date and not needed. What do you think?

Reply:
I have removed the release notes step.